torchaudio/prototype/ctc_decoder/ctc_decoder.py

```diff
@@ -129,20 +129,28 @@ def __call__(self, emissions: torch.FloatTensor, lengths: Optional[torch.Tensor]
 
         Args:
             emissions (torch.FloatTensor): tensor of shape `(batch, frame, num_tokens)` storing sequences of
-                probability distribution over labels; output of acoustic model
+                probability distribution over labels; output of acoustic model. It must live on CPU.
```

Comment:
perhaps just "CPU tensor of shape `(batch, frame, num_tokens)` storing sequences of probability distribution over labels; output of acoustic model."? sounds a little more concrete than "live on CPU"

and so forth below